conu/apidefs/container.py

```diff
@@ -194,12 +194,10 @@ def execute(self, command, **kwargs):
         """
         execute a command in this container -- usually the container needs to be running
 
-        TODO: what about parameters?
-
-        :param command: str, command to execute in the container
-        :param kwargs:
-        :return: ? we need to provide output, exit code and there needs to be a possibility for
-                  this thing to be async and blocking
+        :param command: list of str, command to execute in the container
+        :param exec_create_kwargs: dict, params to pass to exec_create()
```

Comment:
these are not specified in the method definition -- could we list kwargs here and say those will container specific parameters for container engines?